src/fragments/lib/project-setup/ios/upgrade-guide/upgrade-guide.mdx

```diff
@@ -35,18 +47,61 @@ func signIn(username: String, password: String) async throws {
 
 ### Combine support
```

Comment:
Just saw that we have another section for combine the doc - https://docs.amplify.aws/lib/project-setup/combine/q/platform/ios/. Looks like the content below are duplicate